README.md

```diff
@@ -1758,35 +1762,35 @@ the company.
 
 ### Taxes on ISOs and NSOs
 
-Typically, early to mid-stage companies grant stock options, and they may be
+Typically, early to mid-stage companies grant stock options, which may be
 [ISOs or NSOs](#kinds-of-stock-options).
 
-- ❗ **When you owe tax**: When you get stock options and are considering if and when to
-  exercise them, you need to think about the taxes.
-  In principle, you need to think about taxes you may incur at three points in time:
+- ❗When you get stock options and are considering if and when to exercise, you need to think
+  about the taxes and **when you owe them**. In principle, you need to think about taxes you
+  may incur at three points in time:
   - at **time of grant**
   - at **time of exercise**
   - at **time of sale**
-- These events trigger ordinary tax (high), long-term capital gains (low), or AMT (possibly
-  high) taxes in different ways for NSOs and ISOs.
+- These events trigger ordinary tax (high), long-term capital gains (lower), or AMT
+  (possibly high) taxes in different ways for NSOs and ISOs.
   - 🄳 The taxes at time of exercise will depend on the gain between the strike price and the
     FMV, known as the **spread** or the
     [**bargain element**](http://www.investorwords.com/5414/bargain_element.html).
-- 🔹 If you are granted ISOs or NSOs at a low strike price, and the bargain element is zero,
+- 🔹 If you’re granted ISOs or NSOs at a low strike price, and the bargain element is zero,
   then you may be able to exercise at a reasonable price without triggering taxes at all.
-  So assuming the company allows it, it makes sense to early exercise immediately (buying
+  So assuming the company allows it, it makes sense to early exercise *immediately* (buying
   most or all of the shares, even though they’re not vested yet) and simultaneously file an
   83(b) election.
-- 🔸 As we laid out in the previous section, [83(b) elections](#83b-elections) are elections to
-  be taxed on the receipt of property even though you might have to forfeit or give back the
-  property to the company.
+- 🔸 As we laid out in the previous section, an [83(b) election](#83b-elections) is the choice
```

Comment:
An 83(b) election, as already discussed, is ...

(will have mouseover)

Reply:
Fixed.